### PR TITLE
feat(calldirectory): add call directory plugin

### PR DESCRIPTION
**Pull request by NiklasMerz**
This adds the plugin for identfifying incomming calls: https://www.npmjs.com/package/cordova-plugin-call-directory

### CODE REVIEW

File: src/@ionic-native/plugins/call-directory/index.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ * ```
+ */
+@Plugin({
+  pluginName: 'CallDirectory',
+  plugin: 'cordova-plugin-call-directory',
+  pluginRef: 'CallDirectory',
+  repo: 'https://github.com/GEDYSIntraWare/cordova-plugin-call-directory',
+  install: 'cordova plugin add cordova-plugin-call-directory --variable EXT_NAME="Cordova-Directory" --variable ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES="NO"',
+  installVariables: ['EXT_NAME', 'ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'],
+})
```
Comment (danielsogl) [inline src/@ionic-native/plugins/call-directory/index.ts]: add supported platforms (e.g platforms: ['Android', 'iOS'])

File: src/@ionic-native/plugins/call-directory/index.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+   * @return {Promise<boolean>} Returns a promise with result
+   */
+  @Cordova()
+  isAvailable(): Promise<boolean> {
+    return;
+  }
+
+  /**
+   * Add identification numbers
+   * @param items {Array<CallDirectoryItem>} Set of numbers with labels
```
Comment (danielsogl) [inline src/@ionic-native/plugins/call-directory/index.ts]: Change JSDOC to @param {type} param description

File: src/@ionic-native/plugins/call-directory/index.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+   * @return {Promise<any>} Returns a promise that resolves when numbers are added
+   */
+  @Cordova()
+  addIdentification(items: string): Promise<any> {
+    return;
+  }
+
+  /**
+   * Remove identification numbers
+   * @param items {Array<CallDirectoryItem>} Set of numbers with arbitrary label
```
Comment (danielsogl) [inline src/@ionic-native/plugins/call-directory/index.ts]: Change JSDOC to @param {type} param description

**Comment by NiklasMerz** [comment]
Thanks for your feedback. I have corrected those problems. Is adding a separate commit correct or should I squash my changes to one commit?

**Comment by danielsogl** [comment]
I will do the squash merge 😄 You can spam as many commits as you want 😀

**Comment by NiklasMerz** [comment]
I see :smiley: Thanks

**Pull request merged**